src/main/kotlin/uk/gov/justice/digital/hmpps/pollpush/config/LocalstackQueueConfig.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.pollpush.config
+
+import com.amazonaws.services.sqs.AmazonSQS
+import com.amazonaws.services.sqs.model.CreateQueueRequest
+import com.amazonaws.services.sqs.model.QueueAttributeName
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+
+@Configuration
+@Profile("localstack")
+open class LocalstackQueueConfig(
+    private val awsSqsClient: AmazonSQS,
+    @Value("\${sqs.queue.name}") private val queueName: String,
+    @Value("\${sqs.dlq.name}") private val dlqName: String) {
+
+    @Bean
```

Comment:
gone for a java style 4 character indent here - worth ticking the box in intellij to reformat on commit